build-and-test-snapshots.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+set -e
+
+dockerall="./gradlew mysqlbinlogCompose"
+dockercdc="./gradlew mysqlbinlogcdcCompose"
+dockertextsearch="./gradlew mysqlbinlogwithorderhistorytextsearchserviceCompose"
+
+${dockertextsearch}Down
+${dockerall}Down
+${dockercdc}Up
+
+./wait-for-services.sh localhost "8099"
```

Comment:
Not sure that this is needed when gradle is used to start services.

Reply:
I will remove